### DISCUSSION TITLE
pp counter does not use fixed-width numbers

### DISCUSSION THREAD

**Discussion by integerrr** [ppy/osu#15584] [Q&A] [answered]
As opposed to the UR counter, which currently does, is this intended?
Version: 2021.1110.0

https://user-images.githubusercontent.com/7279624/141384053-2daf2ea2-98ea-4395-9d98-db431fd4c5ef.mp4

**Comment by peppy** [comment, marked as answer]
Seems like we use fixed width in most other places, so will change the performance points counter to match.

**Reply by peppy** [reply]
https://github.com/ppy/osu/pull/15587
